src/actions/reject.ts

```diff
@@ -23,4 +24,34 @@ export async function reject(ignoreSslErrors?: boolean) {
 
     core.debug('Deployment strategy selected is Canary. Deleting baseline and canary workloads.');
     canaryDeploymentHelper.deleteCanaryDeployment(kubectl, TaskInputParameters.manifests, includeServices);
+}
+
+export async function rejectBlueGreen(ignoreSslErrors?: boolean) {

```

Comment:
Why'd you choose to have a separate function here?
You could have added the switch on values in the `reject` function right? That way there'd be a single entry point. 
You just call `reject` from the run function.. similar to `deploy` function and it takes care of the strategy and stuff internally.

Reply:
Same would apply for `promote`